packages/atomic/src/components/search/atomic-rating/atomic-rating.pcss

```diff
@@ -1,13 +1,28 @@
 /**
- * @prop --atomic-rating-facet-icon-active-color: Color of the icon when active.
+ * @prop --atomic-rating-icon-active-color: Color of the icon when active.
```

Comment:
IMO, it makes no sense for `facet` to be part of the variable's name since it's not only used in facets.